Utils/Dataflow/run/data4es-start

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash -l
 
-START_OFFSET="09-05-2016 00:00:00"
 BATCH_SIZE=100
-DEBUG=
+DEBUG=1
```

Comment:
Shouldn't this change be mentioned in commit/PR?

Reply:
I´d rather said "it shouldn´t be in the commit at all". Will be fixed, thank you!